terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/el/Functions.java

```diff
@@ -37,7 +34,8 @@
  * <ul>
  * <li>Escaping HTML tag using {@code f:h}</li>
  * <li>Encoding URL using {@code f:u}</li>
- * <li>Replacing new line characters with {@code <br />} using {@code f:br}</li>
+ * <li>Replacing new line characters with {@code <br />
+ * } using {@code f:br}</li>
```

Comment:
@btournc
- [ ] Please make it back to the original.

Reply:
@hirakuriy
 Fixed on below:
 282ea5b53d12a7ca68c5ee75754d69edc920e1c0

Reply:
OK